gNMI_Interface/SubscribeLibrary.py

```diff
@@ -180,14 +185,17 @@ def _wait_on_change_updates(self, config: GNMIConfigTree, timeout: int, update_t
         assert responses, NO_UPDATES
 
     def check_sample_updates(self, period: int, count: int, timeout: int) -> None:
+        '''Check that `count` samples are received in `period`
```

Comment:
"period intervals" sounds confusing - is it number of repetitions, or length of a single one (period count)?

Reply:
The latter - changed the docstring.